spec/support/shared_examples/geocodable_examples.rb

```diff
@@ -14,6 +14,7 @@
 
   def modify_address(subject)
     subject.send("#{address_field_name}=", address_field_updated_value)
+    expect(subject).to have_address_changes
```

Comment:
Let's change this to `have_changed` so we're using the Rails Dirty API directly. `has_address_changes?` could be broken in the model we're testing.